intercom_flutter/pubspec.yaml

```diff
@@ -29,5 +29,5 @@ flutter:
         default_package: intercom_flutter_web
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
Why is this changed?

Reply:
It was automatically corrected by Dart DevTools. It looks like they prefer double quotes. Example from the Flutter site: https://docs.flutter.dev/development/tools/pubspec